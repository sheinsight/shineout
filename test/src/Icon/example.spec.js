import exampleTest from '../../example'

describe('Icon[snapshot]', () => {
  exampleTest('Icon')
})
