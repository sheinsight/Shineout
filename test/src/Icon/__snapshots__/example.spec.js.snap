// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test: Icon[site/pages/components/Icon/example-1-awesome.js] 1`] = `
<div>
  <i />
  <i />
  <i />
  <i />
</div>
`;

exports[`Snapshot test: Icon[site/pages/components/Icon/example-2-iconfont.js] 1`] = `
<div>
  <i>
    î™Ž
  </i>
  <i />
  <i>
    <svg>
      <use />
    </svg>
  </i>
  <i>
    <svg>
      <use />
    </svg>
  </i>
</div>
`;

exports[`Snapshot test: Icon[site/pages/components/Icon/example-3-size.js] 1`] = `
<div>
  <i />
  <i />
  <i />
  <i />
</div>
`;
